File: src/1470-Shuffle-the-Array/shuffle_the_array.js
/**
 * @param {number[]} nums
 * @param {number} n
 * @return {number[]}
 */
var shuffle = function(nums, n) {
  let ans = []

  for (let i = 0; i < n; i++) {
    ans.push(nums[i])
    ans.push(nums[i + n])
  }

  return ans
};

nums = [2,5,1,3,4,7]
n = 3

console.log(shuffle(nums, n))

File: src/1470-Shuffle-the-Array/shuffle_the_array.py
from typing import List

class Solution:
    def shuffle(self, nums: List[int], n: int) -> List[int]:
        ans: List[int] = []

        for i in range(n):
            ans.append(nums[i])
            ans.append(nums[i + n])

        return ans



if __name__ == "__main__":
    nums:  List[int] = [2,5,1,3,4,7]
    n: int = 3

    solution = Solution()

    print(solution.shuffle(nums, n))
    